pvlib/spectrum/mismatch.py

```diff
@@ -0,0 +1,219 @@
+"""
+The ``mismatch`` module provides functions for spectral mismatch calculations.
+"""
+
+import pvlib
+import numpy as np
+import pandas as pd
+from scipy.interpolate import interp1d
+import os
+
+
+def get_example_spectral_response(wavelength=None):
+    '''
+    Generate a generic smooth spectral response (SR) for tests and experiments.
+
+    Parameters
+    ----------
+    wavelength: 1-D sequence of numeric, optional
+        Wavelengths at which the spectral response should be interpolated.
+        By default the wavelengths are from 280 to 1200 in 5 nm intervals. [nm]
+
+    Returns
+    -------
+    sr: pandas.Series
+        The relative spectral response indexed by wavelength in nm. [-]
+
+    Notes
+    -----
+    This spectral response is based on measurements taken on a c-Si cell.
+    The measured data points have been adjusted by PV Performance Labs so that
+    standard cubic spline interpolation produces a curve without oscillations
+    as shown in [1]_, which makes it suitable for experimenting with spectral
```

Comment:
```suggestion
    The measured data points are interpolated using a cubic spline that produces
    a curve without oscillations as shown in [1]_, which makes it suitable for
    experimenting with spectral
```

The phrasing "adjusted by" might invite questions here which may be better left to the reference, unless we add to this note and explain what the adjustment was.